src/output-flow.h

```diff
@@ -26,7 +26,9 @@
 #ifndef __OUTPUT_FLOW_H__
 #define __OUTPUT_FLOW_H__
 
+#include "decode.h"
```

Comment:
You can use `cppclean` to find these cf https://redmine.openinfosecfoundation.org/issues/945